### PR TITLE
feat(install): use shared installation folder by default

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/install/browserPaths.ts [hunk 1]
```diff
@@ -77,9 +77,30 @@ export function executablePath(browserPath: string, browser: BrowserDescriptor):
...
+const defaultBrowsersPath = ((): string | undefined => {
+  const env = getFromENV('PLAYWRIGHT_BROWSERS_PATH');
+  if (env === '0')
+    return undefined;
+
+  let result: string;
+  if (env)
+    result = env;
+  else if (process.platform === 'linux')
+    result = '~/.cache/playwright';
```
Comment (aslushnikov) [inline src/install/browserPaths.ts:89]: Let's do `playwright-nodejs` suffix to isolate from conflicts with native `playwright` application.

File: src/install/browserPaths.ts [hunk 2]
```diff
@@ -77,9 +77,30 @@ export function executablePath(browserPath: string, browser: BrowserDescriptor):
...
+  else if (process.platform === 'darwin')
+    result = '~/Library/Caches/playwright';
+  else if (process.platform === 'win32')
+    result = '~/AppData/Local/playwright';
+  else
+    throw new Error('Unsupported platform: ' + process.platform);
+
+  if (result.startsWith('~'))
```
Comment (aslushnikov) [inline src/install/browserPaths.ts:97]: IIUC the expansion is done by certain shells, but not all. I wouldn't expand this on our side and leave this to the shell.

**Review by aslushnikov — APPROVED** [review]

File: src/install/browserPaths.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/install/browserPaths.ts]: Done

File: src/install/browserPaths.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/install/browserPaths.ts]: Done

**Pull request merged**